packages/auth-providers/dbAuth/web/src/dbAuth.ts

```diff
@@ -49,7 +53,7 @@ export function createDbAuthClient({
   let cachedToken: string | null
 
   const getApiDbAuthUrl = () => {
-    return dbAuthUrl || `${process.env.RWJS_API_URL}/auth`
+    return dbAuthUrl || `${RWJS_API_URL}/auth`
```

Comment:
Why doesn't this one need `globalThis`? Not saying it should or shouldn't, I just don't understand the nuance

Reply:
Thought I replied to this! Not adding the `globalThis` is the only way I can ensure it works with both Webpack and Vite.

I think there's subtle differences in the way these values are defined between the two bundlers